### PR TITLE
Added tooltips for civilian buildings to Tiberian Dawn

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/9067.

### CODE REVIEW

**Comment by pchote** [comment]
Looks good, but you'll need to set up the generic tooltips for in the missions (they should stay as Civilian Building there).

**Comment by atlimit8** [comment]
Needs rebase.

**Comment by Mailaender** [comment]
Rebased. Not sure if the generic tooltip stances are working at all or I just don't understand if they work. Also not sure why they should be generic in single-player.

File: mods/cnc/rules/defaults.yaml [hunk 1]
```diff
@@ -491,7 +491,7 @@
 	Health:
 		HP: 400
 	Tooltip:
-		Name: Civilian Building
+		GenericName: Civilian Building
 		GenericVisibility: None
```
Comment (pchote) [inline mods/cnc/rules/defaults.yaml]: These overrides explicitly disable the generic visibility everywhere...

Comment (Mailaender) [inline mods/cnc/rules/defaults.yaml]: Fixed, but I think I am suffering from yaml merge problems elsewhere. Also tooltips a la "Enemy Civilian Building" isn't really worth it.

Comment (pchote) [inline mods/cnc/rules/defaults.yaml]: We explicitly don't want the "Enemy" / "Allied" for these.  You can turn them off by setting `GenericStancePrefix: false`.

Comment (Mailaender) [inline mods/cnc/rules/defaults.yaml]: Okay, also set ShowOwnerRow to false as well.

**Comment by pchote** [comment]
Still doesn't work in the campaigns:
screen shot 2016-01-03 at 18 17 12

You need to enable the generic visibility in the maps:
```
	^Building:
		Tooltip:
			GenericVisibility: Enemy, Neutral
			ShowOwnerRow: false
```

**Comment by Mailaender** [comment]
Fixed. Note: Don't test with the shroud disable cheats.

**Comment by RoosterDragon** [comment]
In nod01, all the buildings start out as "Civilian Building" but after being destroyed take on one of the new terms like "Pablo's Wicked Pub (Destroyed)". I assume this is not intentional.

**Comment by Mailaender** [comment]
Should be fixed now. It is utter non-sense to me that those should not be displayed in missions and that this exception has to be configured per mission map.yaml.

**Comment by RoosterDragon** [comment]
This still doesn't seem to have fixed the issue.

**Comment by Mailaender** [comment]
Works fine here:

image

You may have tested this with developer cheats on.

**Comment by Mailaender** [comment]
Oh you meant civilian building husks. Fixed that.

**Comment by RoosterDragon** [comment]
:+1:

**Comment by obrakmann** [comment]
You'll need to touch the following maps as well:

* gdi03
* nod03a
* nod03b
* nod05
* nod06a
* nod06b
* nod06c

Other than that, :+1:

**Comment by Mailaender** [comment]
Updated for more missions.

**Comment by abcdefg30** [comment]
Looks good to me, too. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9b225288af95e3082a83d876487e809c74b482f3...62a21f349f42f08bd7c64cdd812270e3786e931c)

**Pull request merged**